platform-android/build.gradle

```diff
@@ -37,8 +34,31 @@ task copyAndroidDependencies {
     }
 }
 
+task copyLicenses {
+    final BASE_PATH = '../sma/'
+    final LIB_PATH = 'src/main/resources/lib'
+    final GO_LICENSE_PATH = BASE_PATH + '3rd-party-licenses/' +
```

Comment:
right now there's just one third-party license but this might change in the future. so I'd suggest recursively copying the `3rd-arty-licenses` folder instead...